flatliners/corrcomparison.py

```diff
@@ -0,0 +1,57 @@
+from .baseflatliner import BaseFlatliner
+
+import numpy as np
+
+class CorrComparisonScore(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+
+        self.score = dict()
+        self.clusters = dict()
+        self.versions = dict()
+
+    def on_next(self, x):
+        """ update l2 distance between cluster vector and baseline vector
+        """
+        # determine entry type
+        entry_type = 'unkown'
+        if str(type(x[list(x.keys())[0]])) == "<class 'pandas.core.frame.DataFrame'>":
```

Comment:
jfyi, 
I think
`isinstance(p, pandas.core.frame.DataFrame)` is way nicer than the string comparison.

Or `type(p).__name__ == 'DataFrame'`

Reply:
And the documentation says:
> The isinstance() built-in function is recommended for testing the type of an object, because it takes subclasses into account.

Reply:
Thanks, I'll change those to `isinstance()`